serviceworker.js

```diff
@@ -0,0 +1,79 @@
+const RUNTIME = 'redditp-cache-v1';
+
+const PRECACHE_URLS = [
+    'https://www.redditp.com',
+    'https://www.redditp.com/index.html',
```

Comment:
It doesn't make sense to precache `https://www.redditp.com/index.html` when that's going to be the resource you get for eg `https://www.redditp.com/r/aww` unless you add clever fetching, which I would also want to avoid because bugs there would be easy to miss and easily break the site.